src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSourceRequest.java

```diff
@@ -187,6 +193,17 @@ protected BitbucketSCMSourceRequest(@NonNull final BitbucketSCMSource source,
         repoOwner = source.getRepoOwner();
         repository = source.getRepository();
         pullRequestData = new HashMap<>();
+        disableNotifications = context.notificationsDisabled();
```

Comment:
Please remove there is already a trait plugin that provides this support: https://plugins.jenkins.io/skip-notifications-trait